components/Text/Text.js

```diff
@@ -3,36 +3,34 @@ import * as styles from "./Text.module.scss";
 export default function Text({ mode, style, children }) {
```

Comment:
```suggestion
export default function Text({ heading = 'h3', size = 'normal', weigth = 'normal', mode, style, children }) {
```

Reply:
This idea is good if we want to customize the text the way we want, but since we have to follow the typography system designed on Figma, this implementation would create a lot of possibilities for developers to make design mistakes.

![Screenshot 2022-12-19 at 14 48 45](https://user-images.githubusercontent.com/41298710/208453215-fc570949-3a45-4210-8b9a-73a406983dbd.png)

### The idea is to set the pattern like this:
(every **HEADING 1**  **HAS** to be **BOLD** and **SIZE = 49px**)